content/docs/concurrent-mode-intro.md

```diff
@@ -14,87 +14,87 @@ next: concurrent-mode-suspense.html
 
-Concurrent Mode techniques reduce the need for debouncing and throttling in UI. Because rendering is interruptible, React doesn't need to artificially *delay* work to avoid stutter. It can start rendering right away, but interrupt this work when needed to keep the app responsive.
+Eşzamanlı Mod tekniği, kullanıcı arayüzlerindeki girdi bekletme ve baskılama ihtiyacını azaltmaktadır. Render işlemi yarıda kesilebilir olduğundan, takılma problemini çözmek için React, sanal bir *gecikmeye* ihtiyaç duymamaktadır. Uygulamanın kullanıcıya olan duyarlılığını korumak için gerektiğinde render işlemini yarıda bırakabilir ve ardından tekrar render etmeye devam edebilir.
 
-### Intentional Loading Sequences {#intentional-loading-sequences}
+### Bilinçli Yükleme Sekansları {#intentional-loading-sequences}
 
-We've said before that Concurrent Mode is like React working "on a branch". Branches are useful not only for short-term fixes, but also for long-running features. Sometimes you might work on a feature, but it could take weeks before it's in a "good enough state" to merge into master. This side of our version control metaphor applies to rendering too.
+Daha önce Eşzamanlı Mod'un, React'in "bir branch üzerinde" çalışıyormuş gibi işlediğinden bahsetmiştik. Branch'ler, kısa süreli çözümlerde olduğu gibi, yapımı uzun sürecek özelliklerde de faydalıdır. Bazı zamanlar, master branch ile birleştirilebilecek duruma gelmesi haftalar alabilecek özellikler üzerinde çalışıyor olabilirsiniz. Versiyon kontrol metaforumuzun bu tarafı da render etmeye uygulanabilir.
 
-Imagine we're navigating between two screens in an app. Sometimes, we might not have enough code and data loaded to show a "good enough" loading state to the user on the new screen. Transitioning to an empty screen or a large spinner can be a jarring experience. However, it's also common that the necessary code and data doesn't take too long to fetch. **Wouldn't it be nicer if React could stay on the old screen for a little longer, and "skip" the "bad loading state" before showing the new screen?**
+Bir app üzerinde iki farklı sayfa arasında geçiş yaptığımızı hayal edin. Bazı durumlarda kullanıcıya yeterli kalitede bir yükleme sekansı veremeyecek kadar kod ya da veri yüklenmemiş olabilir. Boş bir ekrana geçiş yapmak ya da kocaman bir dönen çubuk görmek hoş bir deneyim olmayacaktır. Ancak, gerekli kod ve verinin elde edilmesinin çok uzun sürmemesi de oldukça sık rastlanan bir durumdur. **Böyle bir durumda React eski ekranda biraz daha kalsa ve yeni ekranı göstermeden önce "kötü yükleme sekansını" "pas geçse" iyi olmaz mıydı?**
```

Comment:
Maybe we can use `uygulama` instead of `app`. What do you think? There are several places that can be updated, I believe.